spec/e2e/enum_spec.rb

```diff
@@ -32,10 +33,15 @@
       expect(StoredFile.types).to eq(unknown: 0, image: 1, video: 2)
       expect(StoredFile.sizes).to eq(big: 100, medium: 7, small: 2)
       expect(StoredFile.flags).to eq(clean: 0, dangerous: 1)
+      expect(StoredFile.type_formats).to eq(Mpeg: 0, Png: 1)
       expect(UploaderRel.origins).to eq(disk: 0, web: 1)
     end
   end
```

Comment:
There should be a spec somewhere is this file which tests that the `ArgumentError` gets raised when a non-lowercase option is passed to `enum`.  Probably you can dynamically, in the spec, call `StoredFile.enum something: [:value1, :Value1]`